experimental/generation/generator/src/library/processSchemas.ts

```diff
@@ -98,6 +98,12 @@ export function typeName(property: any): string {
     if (property.enum) {
         type = 'enum'
     }
+
+    if (property.format) {
+        type = property.format
+        delete property.format
```

Comment:
> delete property.format [](start = 7, length = 23)

Do not do this.  We still want the underlying schema to be accurate since it passes through to the end.